utils/R/utils.R

```diff
@@ -665,6 +665,32 @@ convert.expr <- function(expression) {
   return(list(variable.drv = deri.var, variable.eqn = list(variables = variables, expression = deri.eqn)))
 }
 
+## Convert a file using Brown Dog Conversion service
```

Comment:
since this function is commented out, i think it would be better not event to check it in.